v1/capabilities.go

```diff
@@ -1,37 +1,27 @@
 package v1
 
+import "slices"
+
 type Capability string
 
 type Capabilities []Capability
 
 func (c Capabilities) IsCapable(cc Capability) bool {
-	for _, capability := range c {
-		if capability == cc {
-			return true
-		}
-	}
-	return false
+	return slices.Contains(c, cc)
 }
 
 const (
 	CapabilityCreateInstance           Capability = "create-instance"
 	CapabilityCreateIdempotentInstance Capability = "create-instance-idempotent"
 	CapabilityTerminateInstance        Capability = "terminate-instance"
+	CapabilityCreateTerminateInstance  Capability = "create-terminate-instance"
```

Comment:
this verbage is a bit weird, might be worth a comment to explain what this means